scripts/functional_tests.js

```diff
@@ -13,7 +13,6 @@ const alwaysImportedTests = [
   require.resolve('../test/ui_capabilities/newsfeed_err/config.ts'),
   require.resolve('../test/new_visualize_flow/config.ts'),
   require.resolve('../test/security_functional/config.ts'),
-  require.resolve('../test/functional/config.legacy.ts'),
```

Comment:
ℹ️  This functional test suite run the tests for the legacy implementation (angular). The same tests run for the new implementation so we don't lose our test coverage here!